.github/actions/cmake_config/entrypoint.sh

```diff
@@ -10,7 +10,7 @@ rm -rf cmake_dir/* build_tmp/*
 v=$1
 fn=cmake-$v-Linux-x86_64.tar.gz
 
-if [ ! -f cmake_souces/$fn ]; then
+if [ ! -f cmake_sources/$fn ]; then
```

Comment:
This worries me

Reply:
It was an attempt to cache, IIRC, and doesn't really affect anything (other than failing to cache).